sdk/servicebus/service-bus/src/sender.ts

```diff
@@ -80,6 +80,20 @@ export class Sender {
     return sender.send(message);
   }
 
+  /**
+   * Ensures that the Service Bus connection and sender link is open.
+   *
+   * This method will be called on-demand by the Sender. Only call this method if you need
+   * to force the opening of the connection and link.
+   */
+  async open() {
+    // the underlying `MessageSender` initialization will both
+    // cache the sender in the context for the next send and
+    // ensure the link and connection are initialized.
```

Comment:
I'd rather update the docs on the `MessageSender.create()` method to convey that it only creates a new sender if none already exists, than having the docs on the caller side. Otherwise, what is the reason to have this comment only for `open()`, but not for `send()` and `sendBatch()`?

Reply:
In this case because it's not obvious why I'm creating this sender and then throwing it away and expecting it to have any effect.

Within MessageSender, since it's the class doing all of this "background caching" I'd expect people to get that (although I'm perfectly fine with having this comment in both places).

Reply:
Gone.